source/MDIViewMenu.cpp

```diff
@@ -31,6 +31,8 @@
 #include "MDIClientView.h"
 #include "MDIViewMenu.h"
 
+using namespace std;
```

Comment:
Yes. Try removing it :P
On Jan 16, 2015 6:34 PM, "Augustin Cavalier" <notifications@github.com>
wrote:

> In source/MDIViewMenu.cpp
> <https://github.com/HaikuArchives/BeMDI/pull/1#discussion-diff-23095296>:
>
> > @@ -31,6 +31,8 @@
> >  #include "MDIClientView.h"
> >  #include "MDIViewMenu.h"
> >
> > +using namespace std;
>
> Is this really necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HaikuArchives/BeMDI/pull/1/files#r23095296>.
>